src/pykeen/utils.py

```diff
@@ -820,11 +821,15 @@ def upgrade_to_sequence(x: Union[X, Sequence[X]]) -> Sequence[X]:
     >>> upgrade_to_sequence((1, 2, 3))
     (1, 2, 3)
     >>> upgrade_to_sequence("test")
-    ("test",)
+    ('test',)
     >>> upgrade_to_sequence(list("test"))
-    ("t", "e", "s", "t")
+    ('t', 'e', 's', 't')
     """
-    return x if (isinstance(x, Sequence) and not isinstance(x, str)) else (x,)
+    if isinstance(x, str):
+        return cast(Sequence[X], (x,))
```

Comment:
maybe better just to ignore the type on this return

Reply:
this would completely disable the mypy check for this line :-/

Reply:
yes, but we're very sure is right. the way to make MyPy work properly for this is to do some super verbose overloading of the signature

Reply:
e006db8